src/components/Button/index.tsx

```diff
@@ -7,24 +7,55 @@ import styles from './styles.module.css';
 export enum ButtonKind {
   /** A primary button, generally displayed in green. */
   PRIMARY = 'primary',
-  /** An inverted button style, generally displayed in white on a dark background. */
-  INVERTED = 'inverted',
   /** A button that only has a border. */
   OUTLINE = 'outline',
   /** A button which cannot be clicked, generally displayed in grey. */
   DISABLED = 'disabled',
+  /** A button which has no border, fits as a tertirary button. */
+  BORDERLESS = 'borderless',
+  /** A button that dispalys well on image distracting backgrounds */
+  SCREEN = 'screen',
 }
 
 export enum ButtonSize {
-  MEDIUM = 'medium',
-  SMALL = 'small',
+  SM = 'small',
+  MD = 'medium',
+  LG = 'large',
+}
+
+export enum ExtendedPadding {
+  XS = 'padding-xs',
+  SM = 'padding-sm',
+  MD = 'padding-md',
+  LG = 'padding-lg',
+  XL = 'padding-xl',
+}
+
+export enum AlternativeColors {
+  GREEN = 'green',
+  RED = 'red',
+  BLUE = 'blue',
+  YELLOW = 'yellow',
+  GREY = 'safe-grey',
+  PURPLE = 'safe-purple',
+}
+
+export enum HoverBehaviour {
+  LIGHTEN = 'bg-lighten',
+  DARKEN = 'bg-darken',
 }
```

Comment:
I don't doubt that we actually have both these behaviors throughout the site, but I wonder if we should unify on one or the other for consistency? 🤔

Reply:
Thanks for taking the time to review. Yep, we can continue to tailor them down. 

I design them with the intention to bring more headroom for some ordered customizability. The idea was to always use default/fallback values as means to maintain consistency i.e. omitted hoverBehaviour prop will default to 'bg-darken'. Like any optional props here, they are customizable that should be used situationally when needed . 

😅 tho I agree nonetheless, I think more nuance customization do come with more understanding and responsibility to use appropriately. hover-behaviour may just be too much, so yes, unify on one. I'll declutter it, and keep our core options, i.e. size/color/padding/etc. Better to expand accordingly as opposed to over-preparing .